ZuluContent/Zulu/Engines/Magic/HitScripts/TriElementalStrike.cs

```diff
@@ -8,11 +8,12 @@ namespace Server.Engines.Magic.HitScripts
 {
     public class TriElementalStrike : WeaponAbility
     {
+        // TODO convert to ITargetableAsyncSpell
```

Comment:
Guess this is for another PR?

Reply:
Yep, will refactor when convert all the circles are done, need to do the same thing with `SpellHit`